disnake/http.py

```diff
@@ -257,16 +257,14 @@ def recreate(self) -> None:
             )
 
     async def ws_connect(self, url: str, *, compress: int = 0) -> aiohttp.ClientWebSocketResponse:
-        if hasattr(aiohttp, "ClientWSTimeout"):
-            timeout = aiohttp.ClientWSTimeout(ws_close=30.0)  # pyright: ignore[reportGeneralTypeIssues]
-        else:
-            timeout = 30.0
         return await self.__session.ws_connect(
             url,
             proxy_auth=self.proxy_auth,
             proxy=self.proxy,
             max_msg_size=0,
-            timeout=timeout,
+            timeout=aiohttp.ClientWSTimeout(ws_close=30.0)  # type: ignore
+            if hasattr(aiohttp, "ClientWSTimeout")
+            else 30.0,
```

Comment:
this seemed fine before, having a `type: ignore` here has the potential to ignore errors about entirely missing arguments as well

Reply:
This is having issues with being tested across dependency versions that *don't* match.

Reply:
Works for now, reverted. [`44288fc` (#1329)](https://github.com/DisnakeDev/disnake/pull/1329/commits/44288fc0f8dc2eaef5fb4f6fdafd352799aefb2b)